packages/m/memtest86plus/package.yml

```diff
@@ -0,0 +1,33 @@
+name       : memtest86plus
+version    : '7.00'
+release    : 1
+source     :
+    - https://www.memtest.org/download/v7.00/mt86plus_7.00.src.zip : 684fb63a96ef114c4b4a75ed8d8dc8313d7ed2781c13966eb027b7fae8dd9595
+homepage   : https://www.memtest.org
+license    : GPL-2.0
+component  : system.utils
+summary    : Stand-alone memory tester for x86 and x86-64 architecture computers.
+description: |
+    Memtest86+ provides a more thorough memory check than that provided by BIOS memory tests.
+
+    It can be loaded and run either directly by a PC BIOS (legacy or UEFI) or via an intermediate bootloader that supports the Linux 16-bit, 32-bit, 64-bit, or EFI handover boot protocol.
+    
+    It should work on any Pentium class or later 32-bit or 64-bit x86 CPU. 
+builddeps  :
+    - tree
+build      : |
+    pushd build32
+    %make
+    popd
+    pushd build64
+    %make
+    popd
+    tree -l -L 1 build*
+install    : |
+    TARGET="$installdir/usr/libexec/memtest86plus"
+    install -Dm00755 build32/memtest.bin ${TARGET}/build32/memtest.bin
+    install -Dm00755 build32/memtest.efi ${TARGET}/build32/memtest.efi
+    install -Dm00755 build64/memtest.bin ${TARGET}/build64/memtest.bin
+    install -Dm00755 build64/memtest.efi ${TARGET}/build64/memtest.efi
+    cp -v README.md ${TARGET}/
+    cp -vR grub ${TARGET}/
```

Comment:
I think we should either install `grub/grub-legacy.cfg` into `/etc/grub.d/20_memtest86+`, or we shouldn't provide them at all and let CBM handle it.

I'm not familiar enough with the grub side of CBM to know what the correct decision is here, so let's just omit this directory for now.